src/figuresClasses.ts

```diff
@@ -1,11 +1,97 @@
-export interface Figure {}
+enum Color {
+  red = 'red',
+  green = 'green',
+  blue = 'blue',
+}
+
+enum Shape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle',
```

Comment:
Enum constant names should start with a capital letter. Consider changing 'triangle', 'circle', and 'rectangle' to 'Triangle', 'Circle', and 'Rectangle'.